content_resolver/config_manager.py

```
@@ -971,9 +931,7 @@ def get_configs(self):
                 try:
                     json_data = self.load_data(os.path.join(directory, json_file))
                 except:
-                    raise ConfigError("Error loading a JSON data file '{filename}': {err}".format(
-                                    filename=json_file,
-                                    err=err))
+                    raise ConfigError(f"Error loading a JSON data file '{json_file}': {err}")
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)